eclass/distutils-r1.eclass

```diff
@@ -820,6 +820,10 @@ distutils-r1_python_test() {
 		die "${FUNCNAME} can be only used after calling distutils_enable_tests"
 	fi
 
+	if [[ -n "${PYTHONPATH}" && "${PYTHONPATH@a}" != *x* ]]; then
```

Comment:
Could you try checking if `EPYTHON` is set instead? That's the primary variable used to indicate we're in ebuild env.

Reply:
good point. done

Reply:
Cool. Could you cover other similar functions too? (i.e. `distutils-r1_python_compile` and so on)